includes/auto_loaders/config.core.php

```diff
@@ -72,6 +72,8 @@
                                'objectName'=>'zcPassword');
   $autoLoadConfig[0][] = array('autoType'=>'class',
                                'loadFile'=>'Customer.php');
+  $autoLoadConfig[0][] = array('autoType'=>'class',
```

Comment:
i see no reason why you should not use the short array syntax

Reply:
Fair enough, I was trying to fit in with old code but I see zcDate's changes use the new syntax, and tighter indentation. I've updated my syntax.